test/helpers/cilium.go

```diff
@@ -261,22 +288,37 @@ func (c *Cilium) PolicyImport(path string, timeout time.Duration) (int, error) {
 		c.logCxt.Infof("PolicyImport: current revision %d same as %d", currentRev, revision)
 		return false
 	}
-	err = WithTimeout(body, "Couldn't import policy revision", &TimeoutConfig{Timeout: timeout})
+	err = WithTimeout(body, "Could not import policy revision", &TimeoutConfig{Timeout: timeout})
 	if err != nil {
 		return -1, err
 	}
 	revision, err = c.PolicyGetRevision()
-	c.logCxt.Infof("PolicyImport: finish '%s' with revision '%d'", path, revision)
+	c.logCxt.Infof("PolicyImport: finished '%s' with revision '%d'", path, revision)
 	return revision, err
 }
 
-//PolicyWait helper function that executes cilium policy wait so will be executing meanwhile
-// all endpoints are with this policy
-func (c *Cilium) PolicyWait(id int) *CmdRes {
-	return c.Exec(fmt.Sprintf("policy wait %d", id))
+//PolicyWait executes cilium policy wait, which waits until all endpoints are
+//updated to the given policy revision.
+func (c *Cilium) PolicyWait(revisionNum int) *CmdRes {
+	return c.Exec(fmt.Sprintf("policy wait %d", revisionNum))
+}
+
+//ReportFailed Write to log output the needed data to debug an error
```

Comment:
Depending upon the scope (k8s or runtime) we should run CiliumReport / an equivalent function for the runtime tests that gathers all Cilium CLI state. See `tests/helpers.bash:dump_cli_output` and `tests/helpers.bash:dump_cli_output_k8s`.